src/js/view/error-container.ts

```diff
@@ -1,15 +1,28 @@
 import errorContainerTemplate from 'templates/error';
 import { createElement } from 'utils/dom';
 import { style } from 'utils/css';
+import { OS } from 'environment/environment';
+import TizenErrorContainer from 'view/controls/tizen/tizen-error-container';
 import type Model from 'controller/model';
 import type { PlayerError } from 'api/errors';
+import type { PlayerAPI } from 'types/generic.type';
 
-export default function ErrorContainer(model: Model, error: PlayerError): HTMLElement {
+require('css/error.less');
+
+export default function ErrorContainer(api: PlayerAPI, model: Model, error: PlayerError): HTMLElement {
     const { message, code } = error;
     const html = errorContainerTemplate(model.get('id'), message, model.get('localization').errors.errorCode, code.toString());
+    const element = createElement(html);
+    style(element, {
+        background: 'black'
+    });
+
+    if (OS.tizen) {
+        return TizenErrorContainer(api, element, model.get('localization'));
+    }
```

Comment:
I don't like ErrorContainer being modified to return TizenErrorContainer adding the additional api dependency. Maybe the keyboard input could be completely separated from the UI?

The only reason the web player implements key input in controls is because it's only supposed to listen for that input when `controls: true` are enabled. If that's not the case for Tizen let's always have those inputs on the tizen player container.